.github/workflows/deploy.yml

```diff
@@ -522,8 +522,10 @@ jobs:
 
       - name: Install tools
         run: |
-          sudo apt-get update
-          sudo apt-get install -y virtualbox vagrant
+          wget -O- https://apt.releases.hashicorp.com/gpg | sudo gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg
+          echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
```

Comment:
Without these, we get `Package 'vagrant' has no installation candidate` both using `apt` and `apt-get`

Failed runs:
https://github.com/openDAQ/openDAQ/actions/runs/11126132989/job/30917827291
https://github.com/openDAQ/openDAQ/actions/runs/11129401771/job/30928420708

Fix:
https://developer.hashicorp.com/vagrant/downloads#linux